user_guide_src/source/changelogs/v4.6.0.rst

```diff
@@ -277,6 +277,9 @@ Deprecations
     - The properties ``$arguments`` and ``$argumentsClass`` of ``Filters`` have
       been deprecated. No longer used.
     - The ``Filters::getArguments()`` method has been deprecated. No longer used.
+- **File:**
+    - The function ``getSizeByUnit()`` of ``File`` has been deprecated.
+      Use either ``getSizeByBinaryUnit()`` or ``getSizeByMetricUnit()`` instead.
```

Comment:
Please add the new methods in the **Enhancements** section:
https://github.com/codeigniter4/CodeIgniter4/blob/4.6/user_guide_src/source/changelogs/v4.6.0.rst#libraries

Reply:
Will do. Two separate bullet point, or both in one?

Reply:
One line would be sufficient.